cfpb/design-manual
# Updates the project assets to use the latest release from Capital Framework.

Updates the project to use the latest release from Capital Framework.

## Changes

- ~~Updates cf-expandables v7.0.0~~
- ~~Updates cf-atomic-component to v2.0.0~~
- ~~Updates cf-tables to v5.0.0~~
- Adds initialization code for cf-expandables and cf-tables.
- Regenerates assets with `grunt`.

(Added by Ans)

## Testing

- Run `npm start`
- Open http://localhost:4000/page-components/expandables.html and make sure the expandables continue to work
- Open http://localhost:4000/page-components/tables.html and make sure the tables continue to work

## Notes

We'll need decide what to do about accordions documentation. Turns out they are necessary for the OaH explanations, but I don't know how much we want to encourage their use outside of those pages. The simplest answer is to probably revert this change https://github.com/cfpb/design-manual/commit/bb035a7ca5245d8093bcda54b3269044e28fe331#diff-7c1ba7f64a56bc337d12c911525c729f but I'll leave that up to the design team.
